WooCommerce/Classes/ViewRelated/Orders/Order Details/Shipping Labels/WooShipping Create Shipping Labels/WooShippingAddresses/WooShippingEditAddressViewModel.swift

```diff
@@ -174,6 +174,14 @@ final class WooShippingEditAddressViewModel: ObservableObject, Identifiable {
     /// Error from remote validation, if any.
     @Published private var remoteValidationError: String?
 
+    /// Current address update error alert state
+    @Published var isShowingAddressErrorAlert = false
+    @Published var addressErrorState: AddressErrorState = .none {
```

Comment:
Nit: this property doesn't need to be published. It's sufficient for the view to observe `isShowingAddressErrorAlert` to show the alert. Since the state doesn't need to be updated from outside, keeping it read-only is necessary too.

```suggestion
    private(set) var addressErrorState: AddressErrorState = .none {
```